src/redturtle/prenotazioni/profiles/default/metadata.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <metadata>
-  <version>1501</version>
+  <version>1502</version>
```

Comment:
maybe it's better an higher step..1600 for example

Reply:
Why

Reply:
I mean we are under the *.dev version and do not even use the major release

Reply:
but this will be a big change and we shouldn't be under the dev version for me

Reply:
solved